src/index.html

```diff
@@ -7,12 +7,81 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <a href="#">
+          <img
+            class="card__image-content"
+            src="../src/images/imac.jpeg"
+            alt="content"
+          />
+        </a>
+      </div>
+
+      <div class="card__info">
+        <h3 class="card__info-text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+      </div>
+
+      <div class="card__code">
+        <span class="card__code-text">Product code: 195434</span>
```

Comment:
```suggestion
        <p class="card__code-text">Product code: 195434</p>
```
It's better to use semantic tag for the text